test/box/fix-not-deleting-unix-socket-path.result

```diff
@@ -0,0 +1,94 @@
+-- test-run result file version 2
+env = require('test_run')
+ | ---
+ | ...
+net_box = require('net.box')
+ | ---
+ | ...
+fio = require('fio')
+ | ---
+ | ...
+test_run = env.new()
+ | ---
+ | ...
+test_run:cmd("create server test with script='box/fix-not-deleting-unix-socket-path.lua'")
+ | ---
+ | - true
+ | ...
+
+-- Check that unix socket path deleted after tarantool is finished
+test_run:cmd("setopt delimiter ';'")
+ | ---
+ | - true
+ | ...
+for i = 1, 2 do
+    local thread_count = i
+    test_run:cmd(string.format("start server test with args=\"%s\"", thread_count))
+    server_addr = test_run:cmd("eval test 'return box.cfg.listen'")[1]
+    test_run:cmd("stop server test")
+    assert(fio.path.exists(server_addr) == false)
+end
+test_run:cmd("setopt delimiter ''");
+ | ---
+ | ...
+
+-- Check, that all sockets are closed correctly,
+-- when the listening address is changed.
+test_run:cmd(string.format("start server test with args=\"%s\"", 2))
```

Comment:
You could also inline '2' into the string and then you wouldn't need `string.format`.

Reply:
Fixed